BGAnimations/ScreenEvaluation common/Shared/AutoSubmitScore.lua

```diff
@@ -119,11 +119,11 @@ local AutoSubmitRequestProcessor = function(res, overlay)
 						end
 					end
 
-					-- Only display the RPG on the sides that are actually joined.
-					if ToEnumShortString("PLAYER_P"..i) == "P"..side and data[playerStr]["rpg"] then
-						local rpgAf = overlay:GetChild("AutoSubmitMaster"):GetChild("RpgOverlay"):GetChild("P"..i.."RpgAf")
-						rpgAf:playcommand("Show", {data=data[playerStr]["rpg"]})
-						hasRpgData = true
+					-- Only display the overlay on the sides that are actually joined.
+					if ToEnumShortString("PLAYER_P"..i) == "P"..side and (data[playerStr]["rpg"] or data[playerStr]["itl"]) then
```

Comment:
Not wrong, just a stylistic comment: I usually try to use the attribute syntax instead of brackets when the key is a constant string, e.g. `data[playerStr].itl` instead of `data[playerStr]["itl"]`.

Reply:
I've been using brackets specifically for the request content because it feels more like I'm "keying into the JSON request". I think for other parts of the codebase (say accessing a song hash, I do use the dot notation). Thoughts?

Reply:
I would always use the attribute syntax when possible, just because it's the same thing and easier to type. Not a big deal though.